src/hamburger.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const hamburgerBtn = document.querySelector('.nav__hamburger-button');
+  const menu = document.querySelector('.nav__menu');
+  const menuLinks = document.querySelectorAll('.nav__link');
+
+  const toggle = () => {
+    menu.classList.toggle('nav__menu-display');
+    hamburgerBtn.classList.toggle('nav__hamburger-clicked');
+  };
+
+  menuLinks.forEach((element) => {
+    element.addEventListener('click', () => {
```

Comment:
You can omit this anonymous function here. Instead:
``` javascript
el.method('whatever', () => {
  toggle();
});
```
you can choose:
``` javascript
el.method('whatever', toggle);
```
The second syntax will be shorter and if the method toggle will throw an exception you will have a more meaningful stack trace.

PS, In some cases you will have to use `.bind()` on handler. (In example: `toggle.bind(this)`).